src/components/pipelineRuns-list/PipelineRunsList.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  ListPageBody,
+  ListPageFilter,
+  VirtualizedTable,
+  getGroupVersionKindForModel,
+  useK8sWatchResource,
+  useListPageFilter,
+} from '@openshift-console/dynamic-plugin-sdk';
+import * as React from 'react';
+import usePipelineRunsColumns from './usePipelineRunsColumns';
+import { usePipelineRunsFilters } from './usePipelineRunsFilters';
+import { PipelineRunKind } from '../../types';
+import { PipelineRunModel } from '../../models';
+import { useGetTaskRuns } from '../hooks/useTektonResult';
+import PipelineRunsRow from './PipelineRunsRow';
+import { useTranslation } from 'react-i18next';
+
+import './PipelineRunsList.scss';
+import { useParams } from 'react-router-dom-v5-compat';
+
+type PipelineRunsListProps = {
+  namespace: string;
+};
+
+const PipelineRunsList: React.FC<PipelineRunsListProps> = ({ namespace }) => {
+  const { t } = useTranslation();
+  const { ns } = useParams();
+  namespace = namespace || ns;
+  const columns = usePipelineRunsColumns(namespace);
+  const filters = usePipelineRunsFilters();
+  const [pipelineRuns, pipelineRunsLoaded, pipelineRunsLoadError] =
+    useK8sWatchResource<PipelineRunKind[]>({
+      isList: true,
+      groupVersionKind: getGroupVersionKindForModel(PipelineRunModel),
+      namespace,
+      optional: true,
+    });
```

Comment:
https://github.com/openshift-pipelines/console-plugin/assets/102503482/fc498aff-5c99-40cd-9fa7-8abb81014bf2

PLR from tekton results is not fetched.